wpilibc/.styleguide

```diff
@@ -0,0 +1,3 @@
+generatedFileExclude {
+  wpilibc/src/generated
```

Comment:
Move this to the base .styleguide and delete this file. This file is overriding the base file, and it's missing a bunch of stuff.